dynamodb/installer.js

```diff
@@ -3,14 +3,14 @@
 var tar = require('tar'),
     zlib = require('zlib'),
     path = require('path'),
-    http = require('http'),
+    https = require('https'),
     fs = require('fs'),
     ProgressBar = require('progress'),
     utils = require('./utils');
 
 var download = function (downloadUrl, installPath, callback) {
     console.log("Started downloading Dynamodb-local. Process may take few minutes.");
-    http.get(downloadUrl, function (response) {
+    https.get(downloadUrl, function (response) {
```

Comment:
@muriloamendola 

Please resolve this part of the conflict.

Reply:
Sorry, master branch seems to have been modified in many other places besides this one.

Reply:
It appears the original developer may not be active on this thread, I created a applied these same changes on the master branch and created a new PR: [https://github.com/99x/dynamodb-localhost/pull/78]()

Reply:
> Parece que o desenvolvedor original pode não estar ativo neste tópico, criei e apliquei essas mesmas alterações no branch master e criei um novo PR: https://github.com/99x/dynamodb-localhost/pull/78

Amigo como faço para atualizar no meu projeto e voltar a funcionar?

Reply:
Pls we need this PR merged soon!

Reply:
> > Parece que o desenvolvedor original pode não estar ativo neste tópico, criei e apliquei essas mesmas alterações no branch master e criei um novo PR: #78
> 
> Amigo como faço para atualizar no meu projeto e voltar a funcionar?

To get his working we will need the PR merged and the subsequent package [serverless-dynamodb-local](https://github.com/99x/serverless-dynamodb-local) updated to use the new version of this package. I assume the package will be incremented to 0.0.10 when it is merged.

Reply:
FYI: This package (`dynamodb-localhost`) doesn't appear to be maintained, with no new releases to NPM in over 4 years.

**Solution:** You can use [aws-dynamodb-local](https://www.npmjs.com/package/aws-dynamodb-local), a maintained fork, instead. (Disclaimer: I am a contributor to this fork). It is a drop-in replacement for this package, and is updated to fix this bug. If you're using `serverless-dynamodb-local`, you can also replace that with it's equivalent [serverless-dynamodb](https://github.com/raisenational/serverless-dynamodb#migrating-from-serverless-dynamodb-local).

Migrating takes about 2 minutes, with [a full guide in the README](https://github.com/raisenational/aws-dynamodb-local#migrating-from-dynamodb-localhost). Of course, it's all still [open-source](https://github.com/raisenational/aws-dynamodb-local) and MIT licensed. Ownership of this new package sits with a registered charity, that is committed to maintaining the package into the future and is open to contributions from the community.